src/tree/CosmosDBAccountProvider.ts

```diff
@@ -54,12 +54,13 @@ export class CosmosDBAccountProvider implements IChildProvider {
             ],
             wizardContext);
 
+        // https://github.com/Microsoft/vscode-azuretools/issues/120
```

Comment:
I don't understand exactly what this is fixing.

Reply:
Or at least how it relates to https://github.com/Microsoft/vscode-azuretools/issues/120

Reply:
This fixes "Cannot read property 'properties' of undefined" when actionContext is undefined. The reason its undefined is because its not being passed in to createChild as described in that issue

Reply:
But it's being passed as a dummy, so we won't be getting the proper telemetry, right?

Reply:
Yes that is correct. That's why I filed the issue in the shared package, but I never felt like a few missing telemetry properties was high enough priority to work on (Remember - this only happens if people run the command from the command palette)